src/PatientChecking/PatientChecking/Feature/Patient/Queries/GetPatientInDetailByIdQuery.cs

```diff
@@ -27,15 +27,15 @@ public GetPatientInDetailByIdQueryHandler(IPatientService patientService, IAppCo
         }
         public async Task<PatientDetailViewModel> Handle(GetPatientInDetailByIdQuery request, CancellationToken cancellationToken)
         {
-            var cities = await _appConfigurationService.GetProvinceCities();
+            var cities = await _appConfigurationService.GetProvinceCitiesAsync();
             var cityList = new List<string>();
 
             foreach (PatientCheckIn.DataAccess.Models.ProvinceCity p in cities)
```

Comment:
this LOC can be replaced with fluent expression as following:
var cities = (await _appConfigurationService.GetProvinceCitiesAsync()).Select(x=>x.ProvinceCityName);
A better approach I would prefer to do is to only select ProvincityName column from DbContext instead, so application performance will be optimized.

Reply:
I have changed this in code as per your comment. Please review this @minhphien